src/oscar/apps/order/abstract_models.py

```diff
@@ -134,6 +134,13 @@ def set_status(self, new_status):
     set_status.alters_data = True
 
     @property
+    def status_verbose_name(self):
+        status_verbose_names = getattr(settings, 'OSCAR_LINE_STATUS_VERBOSE_NAMES', {})
+        if self.status and status_verbose_names and self.status in status_verbose_names:
```

Comment:
You don't really need the `.. and status_verbose_names ..` do you? The condition wouldn't error if `status_verbose_names` was an empty dict.